### PR TITLE
restrict lit to version 2.3.1

### PR DESCRIPTION
**Pull request by rakeshkumar1019**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -101,7 +103,9 @@ module.exports = (config) => {
     browserConsoleLogOptions: {
       level: 'error',
     },
-    logLevel: config.LOG_WARN,
+    logLevel: config.LOG_DEBUG,
```
Comment (richardsd) [inline karma.conf.js:106]: After everything is fixed, you probably want to reset the log level to `LOG_WARN`.

File: karma.conf.js [hunk 2]
```diff
@@ -101,7 +103,9 @@ module.exports = (config) => {
...
+    captureTimeout: 180000,
```
Comment (richardsd) [inline karma.conf.js:107]: Do we really need such a high value for the `captureTimeout`, or is it just for debugging the problem?

**Review by richardsd — COMMENTED** [review]
It looks good to me.

File: karma.conf.js (hunk 2 above)
Comment (rakeshkumar1019) [reply karma.conf.js:107]: @richardsd thank you for the comment.this is for debugging the problem.

**Pull request closed without merging**